src/ota.rs

```diff
@@ -183,8 +238,8 @@ impl<'a> EspOtaUpdate<'a> {
         Ok(())
     }
 
+    /// No-op: the OTA update is aborted when `EspOtaUpdate` is dropped.
```

Comment:
This consumes `self` without borrowing, forcing `EspOtaUpdate` to be dropped when the function is called.

The comment might be better as `Cancel the update`